README.md

```diff
@@ -1 +1,31 @@
-# oop-school-library
\ No newline at end of file
+## School Library ## 
+
+The objective of this project is to practice Object Oriented Programming. The app will be a library that simulates the following actions:
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+Built With
+
+- Ruby
+
```

Comment:
Please add instructions to run the project locally for a non-technical user.

Reply:
Hi @SajjadAhmad14, a readme file is not even required in the specifications.

Reply:
- [ ] Hi @mateo951 , a professional repo with descriptive readme is part of the general requirements. Kindly refer back to the [project's requirement documentation page](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md) and find the link to the [professional repo documentation page](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) under the "General Requirements" section. Kindly make changes here as per the request made by the previous reviewer. :pray: